tensorflow/python/framework/python_op_gen.cc

```diff
@@ -710,14 +710,24 @@ string GetPythonOp(const OpDef& op_def, const string& function_name) {
 }
 
 string GetPythonOps(const OpList& ops, const std::vector<string>& hidden_ops,
-                    bool require_shapes) {
+                    const string source_file_name, bool require_shapes) {
```

Comment:
In general, please use `const string&` instead of `const string` in function arguments.